modules/cpp-httplib/0.7.15/patches/add_build_file.patch

```diff
@@ -0,0 +1,8 @@
+--- /dev/null
++++ BUILD.bazel
+@@ -0,0 +1,5 @@
++cc_library(
++    name = "httplib",
```

Comment:
Could you add a `cpp-httplib` alias so that users can consume this as `@cpp-httplib`?